wiki/Manual.md

```diff
@@ -5953,109 +5953,26 @@ class BaseError (Exception):
 class BaseError extends \Exception {}
 ```
 
-Below is an outline of exception inheritance hierarchy:
+Here is an outline of exception inheritance hierarchy: https://github.com/ccxt/ccxt/blob/master/ts/src/base/errorHierarchy.ts
```

Comment:
@ttodua let's keep the chart here, it's easier to see how the exceptions are organize without having to open a new link

Reply:
restored